_re2.cc

```diff
@@ -259,15 +284,7 @@ static PyTypeObject Match_Type2 = {
   0,                           /*tp_iternext*/
   match_methods,               /*tp_methods*/
   0,                           /*tp_members*/
-  0,                           /*tp_getset*/
-  0,                           /*tp_base*/
```

Comment:
Why remove these?

Reply:
As above, we no longer have the attr_dict so I trimmed the PyType object after the last member we manually set.

Reply:
Can you leave the zeroes?  I think explicit is better than implicit in this case.

Reply:
Ack